docs/tutorials/tabular_prediction/tabular-quickstart.md

```diff
@@ -10,15 +10,16 @@ import autogluon as ag
 from autogluon import TabularPrediction as task
 ```
 
-Load training data from a CSV file into an AutoGluon Dataset object. This object is essentially equivalent to a [Pandas DataFrame](https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.DataFrame.html) and the same methods can be applied to both.
+Load training data from a [CSV file](https://en.wikipedia.org/wiki/Comma-separated_values) into an AutoGluon Dataset object. This object is essentially equivalent to a [Pandas DataFrame](https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.DataFrame.html) and the same methods can be applied to both.
 
 ```{.python .input}
 train_data = task.Dataset(file_path='https://autogluon.s3.amazonaws.com/datasets/Inc/train.csv')
-train_data = train_data.head(500) # subsample 500 data points for faster demo
+subsample_size = 500 # subsample subset of data for faster demo, try setting this to much larger values
+train_data = train_data.head(subsample_size)
```

Comment:
should we do sample with fixed seed instead of head? Its a better practice in general.